src/routes/(console)/organization-[organization]/usage/[[invoice]]/+page.svelte

```diff
@@ -259,6 +259,63 @@
         </svelte:fragment>
     </CardGrid>
 
+    <CardGrid>
+        <Heading tag="h6" size="7">Image Transformations</Heading>
+
+        <p class="text">
+            The total number of image transformations across all projects in your organization.
+        </p>
+        <svelte:fragment slot="aside">
+            {#if data.organizationUsage.imageTransformationsTotal}
+                <div style:margin-top="-1.5em" style:margin-bottom="-1em">
+                    <BarChart
+                        options={{
+                            yAxis: {
+                                axisLabel: {
+                                    formatter: formatNum
+                                }
+                            }
+                        }}
+                        series={[
+                            {
+                                name: 'Transformations',
+                                data: [
+                                    ...(data.organizationUsage.imageTransformations ?? []).map(
+                                        (e) => [e.date, e.value]
+                                    )
+                                ]
+                            }
+                        ]} />
+                </div>
+
+                <Legend
+                    legendData={[
+                        {
+                            name: 'Transformations',
+                            value: data.organizationUsage.imageTransformationsTotal
+                        }
+                    ]} />
```

Comment:
We don't need this because there's only one data type, unlike Database's Reads & Writes which are 2 distinct legends.